superset/tasks/schedules.py

```diff
@@ -368,15 +391,17 @@ def schedule_email_report(
         schedule.id = schedule_id
         schedule.recipients = recipients
 
-    if report_type == ScheduleType.dashboard.value:
+    if report_type == ScheduleType.dashboard:
```

Comment:
This used to parse around the value (str) of the enum as opposed to the enum instance which seems somewhat atypical.